bundle/compliance/cis_gcp/rules/cis_1_14/rule.rego

```diff
@@ -0,0 +1,25 @@
+package compliance.cis_gcp.rules.cis_1_14
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+default is_valid = false
+
+restrictions if {
+	data_adapter.resource.data.restrictions
+}
+
+is_valid if {
```

Comment:
what happens if the `restrictions` or `apiTargets` fields do not exist?
Is this a valid scenario?

Reply:
> Each key should have a line that says restrictions: followed by varying parameters and NOT have a line saying - service: cloudapis.googleapis.com

If I understand correctly the rule, we need to verify that at least a single restriction exists and verify that it is not the `- service: cloudapis.googleapis.com`